config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Is this file in the root folder? If so are we bundling this folder with our installers? Could this be in somewhere else?